docs/components/tracking_script.rst

```diff
@@ -1,8 +1,442 @@
-Tracking script
-###############
-
-Todo: include MauticJS.
-    Add MauticJS.beforeFirstEventDelivery
-Todo: update to include events
-    mauticPageEventDelivered
-    eventAddedToMauticQueue
\ No newline at end of file
+.. vale off
+
+Tracking script MauticJS
+########################
+
+.. vale on
+
+Mautic provides a means for Plugins to inject custom JavaScript into ``mtc.js``, the PHP generated script that manages Mautic's tracking pixel, Dynamic Web Content, etc.
+You can embed ``mtc.js`` in third party websites to manage communication between those and Mautic.
```

Comment:
The only thing I did for the `MauticJS` docs is to port everything over to RST and fix Vale-related errors/warnings/suggestions. I haven't actually tested the code samples, but `MauticJS` doesn't really change a lot so most of it (if not everything) should still work.